mysql-test/include/table_encrypt_1.inc

```diff
@@ -0,0 +1,277 @@
+# InnoDB transparent tablespace data encryption
```

Comment:
auth_sec.key_rotation_qa - There is quite a number of keyring specific stuff in this test. For instance for keyring_file I delete keyring file to check that keys stay in memory - which will not be possible for keyring_vault. I recreate this file, check for existence etc. I could do it but it will be quite fragmented with includes in different places and maybe quite hard to read	
rpl.rpl_key_rotation – similar situation, lots of checking on keyring file on master and on slave
keyring_udf – those test are already compose of many includes which group common tests for different plugins